src/sage/doctest/reporting.py

```diff
@@ -497,22 +501,21 @@ def report(self, source, timeout, return_code, results, output, pid=None):
                     if output:
                         log("Tests run before doctest exception:\n" + output)
                         log("*"*70)
-                    postscript['lines'].append(cmd + "  # %s" % fail_msg)
+                    postscript['lines'].append(self.report_head(source, fail_msg))
                     if hasattr(result_dict, 'tb'):
                         log(result_dict.tb)
                     if hasattr(result_dict, 'walltime'):
                         stats[basename] = {"failed": True, "walltime": wall, "ntests": ntests}
                     else:
                         stats[basename] = {"failed": True, "walltime": 1e6, "ntests": ntests}
-                    self.error_status |= 64
+                    if not baseline.get('failed', False):  # e.g. AlarmInterrupt in doctesting framework
+                        self.error_status |= 64
```

Comment:
The comment is hard to understand. How about
```python
# We may get here by e.g. AlarmInterrupt in doctesting framework
if not baseline.get('failed', False):
    self.error_status |= 64
```
?

Reply:
Thanks; improved in f61cb9a17bc

Reply:
Sorry for nitpicking. But one reason why the comment is hard to understand is that the comment is inside the if statement. The comment is about the whole if statement, I think.

Reply:
We add a comment for the thing which is below or to the left. No? (Sometimes what I take as granted is not really granted...)

Reply:
You are right, I've fixed it in cf9024f816c